docs/deploy.rst

```diff
@@ -179,13 +179,8 @@ pushing to production.
 
   * Look at the changes to be pushed (`What's Deployed on Kuma`_, and
     `What's deployed on KumaScript`_). To enlist the help of pull request
-    authors and others, you can report bug numbers and PRs in IRC.
-    ``firebot`` will give handy links to Bugzilla.
+    authors and others, you can report bug numbers and PRs in Matrix.
   * Think about manual tests to confirm the code changes work without errors.
-  * Monitor the push in the ``#mdndev`` IRC channel. The final messages
-    (one for kuma, one for kumascript) look like::
-
-        🎉 SUCCESS: Check Rollout Status: Branch stage-push build #104:
```

Comment:
This could be replaced with:
```
  * Monitor the push in the ``#mdn-infra`` Slack channel. The final messages
    (one for kuma, one for kumascript) look like::

        🎉 SUCCESS Check Rollout Status
        🎉 SUCCESS Record Rollout
```